packages/admin-ui-docs/content/button/design.mdx

```diff
@@ -1,279 +1,256 @@
-For example, a save button can be disabled when no changes have been made.
-
-**What to avoid:**
-
-- **Don't display a tooltip when hovering a disabled button.** Maintain the button enabled so when it is clicked you can explain why the action is not possible at the moment.
-- **Don't include an interactive element that is always disabled.** Review the information architecture and reconsider using the component to communicate this context.
-
-**When should an action component trigger a confirmation modal?**
-
-- When the action is irreversible. For example, a delete action where the content cannot be restored from the trash.
-- When the action would take considerable time to undo. For example, a publish action where the publication cannot be canceled while in progress.
-
-</Section>
-
-<Section hasExplanation>
-
-### Content
-
-Learn more about how to craft precise labels to improve button's UX writing.
-
-**What _labels_ and _icons_ should be applied to an action component?**
-Whenever possible, the labels and icons listed below should be adopted. If you need a new label and/or icon please open a request explaining your use case.
-
-| Icon              | Label      | Use case                                                     |
-| ----------------- | ---------- | ------------------------------------------------------------ |
-| DotsThreeVertical | Actions    | Menu with all actions related to the context where it is at. |
-| DotsThreeVertical | More       | Menu after a group of buttons.                               |
-| PlusCircle        | Create new | Button for creating new entities in a database.              |
-| ...               | ...        | ...                                                          |
-
-**When should the _label_ in an action component be hidden?**
-
-- When the same button is repeated multiple times on a screen, usually on lists. An example of this can be found in the Table component, where the “Actions” button is repeated in all rows.
-- When an action is common to a degree where the icon is easily understood by itself. Examples of this include the "Close” button in the Alert component and the "Forward" and "Back” buttons in the Pagination component.
-
-**What to avoid:**
-
-- Including actions that are not always easily understood inside space-constrained elements.
-- Using a tooltip to display their label.
-
-> 💡All actions must have labels to be accessible, even if they are not visible in the interface. The label can be added in the aria-label HTML attribute, so remember to specify what this value should be.
-
-**When should an action component use an _icon_?**
-
-- When the action label is hidden according to our recommendations.
-- When the actions are within the dropdown of the menu component.
+<details>
```

Comment:
What's the difference between this change to the Button and this other one: https://github.com/vtex/admin-ui/pull/444

Reply:
There's no difference, in fact I'm not sure why these commits were replicated here, my mistake 😕

Reply:
Should I revert these changes? I'm afraid when the PR is merged it will roll back the changes applied in https://github.com/vtex/admin-ui/pull/444. Let me know what you think is best @lucasaarcoverde

Reply:
> in fact I'm not sure why these commits were replicated here
You have probably created this branch from the button documentation instead of the `main` branch. It is not critical and it happens, I'll solve the conflict to you and it will be ready to merge